docs/resources/apikey.md

```diff
@@ -0,0 +1,41 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "temporalcloud_apikey Resource - terraform-provider-temporalcloud"
+subcategory: ""
+description: |-
+  Provisions a Temporal Cloud API key.
+---
+
+# temporalcloud_apikey (Resource)
+
+Provisions a Temporal Cloud API key.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `display_name` (String) The display name for the API key.
+- `expiry_time` (String) The expiry time for the API key in ISO 8601 format.
+- `owner_id` (String) The ID of the owner to create the API key for.
+- `owner_type` (String) The type of the owner to create the API key.
+
```

Comment:
please add [disabled](https://github.com/temporalio/api-cloud/blob/7887a9cf05d64dd598dc0054f99b815ef46e9517/temporal/api/cloud/identity/v1/message.proto#L182) to the definition to enable/disable the key via TF